pkg/config/autodiscover/podset_info.go

```diff
@@ -41,13 +37,13 @@ var (
 	}
 )
 
-// DeploymentList holds the data from an `oc get deployments -o json` command
-type DeploymentList struct {
-	Items []DeploymentResource `json:"items"`
+// Deploymen/statefulsettList holds the data from an `oc get /statefulset -o json` command
```

Comment:
I think there's a mistake in this comment, shouldn't it be:

// PodSetList holds the data from an `oc get deployment/statefulset -o json` command

?